roles/contiv/tasks/netplugin_iptables.yml

```diff
@@ -23,7 +23,36 @@
   notify: Save iptables rules
 
 - name: Netplugin IPtables | Open vxlan port with iptables
-  command: /sbin/iptables -I INPUT 1 -p udp --dport 8472 -j ACCEPT -m comment --comment "vxlan"
+  command: /sbin/iptables -I INPUT 1 -p udp --dport 8472 -j ACCEPT -m comment --comment "netplugin vxlan 8472"
```

Comment:
@srampal I would think using the `os_firewall` module would be better suited for something like this.
See https://github.com/openshift/openshift-ansible/tree/master/roles/os_firewall